server/zanata-frontend/src/frontend/app/styles/style.less

```diff
@@ -3756,6 +3768,28 @@ button.success.btn.btn-default:active {
    background-color: @brand-unsure;
  }
 
+#activity-table td button.btn-primary:focus,
+#activity-table td button.btn-info:focus,
+#activity-table td button.btn-warning:focus,
```

Comment:
@aeng, needed to call the button warning, not unsure so the react-bootstrap button component does not return an unknown prop error